content/authentication/troubleshooting-ssh/host-key-verification-failed.md

```diff
@@ -0,0 +1,18 @@
+---
+title: Host key verification failed
+intro: 'As a security precaution, ssh keeps track of which hosts it has previously seen.'
+versions:
+  fpt: '*'
+  ghec: '*'
+topics:
+  - SSH
+shortTitle: Host key verification failed
+---
+
+This means that the key that the server to which your client is connecting presented a key that doesn't match the keys that your client has seen in the past.
+
+It could be because the server changed its keys unexpectedly -- this has happened -- in which case there should be a news reports from trustworthy sources (including the server's official blog) announcing the change.
+
```

Comment:
I think it's worth mentioning the case where the server hasn't actually recently changed, but the key isn't in your known hosts file for whatever reason. Again we can keep this GitHub focused.

```suggestion

You can find an up-to-date list of {% data variables.product.company_short %}'s public SSH keys on {% data variables.product.prodname_docs %}. You may need to add these keys to your `~/.ssh/known_hosts` file. For more information, see "[AUTOTITLE](/authentication/keeping-your-account-and-data-secure/githubs-ssh-key-fingerprints)."

```

Reply:
I should note that `~/.ssh/known_hosts` is _mostly_ a Unix-ism. But I don't have the energy to think about an alternative suggestion (this is mostly a problem for Windows where `~` is mostly not a thing).

I'm going to take this suggestion, and basically each suggestion where I added 👍 and 👀 -- as a pair the 👀 are a reminder to me to think about and the 👍 is a thing I like -- once I've incorporated a thing, I'll remove the 👀.

Reply:
That's a good point, feel free to simplify to "`known_hosts` file"! Id we wanted to make that change then I think the article we're linking to should do the heavy lifting of explaining the file's location across different OSs